languages/kotlin/exercises/concept/basics/.docs/introduction.md

```diff
@@ -0,0 +1,94 @@
+Kotlin is **statically typed** programming language developed by JetBrains. This means that type of variables is defined on compile-time.
+
+## Variables
+
+Similarly to other statically-typed programming languages type of each variable should be defined in compile time. Similarly to other **modern** programming languages you can avoid explicit type declarations where they can be interfered by compiler from context. Use this ability to make your code more readable. But remember that there are cases where it's better to specify type explicitly - e.g. in public APIs.
+
+Kotlin has immutable (`val`) and mutable (`var`) variable. Immutable variable means that once it has it's value assigned - it can't be changed. Most of the time you will use this type of variable.
+
+```kotlin
+val robotName = "HAL-9000"
+userId = "T-1000"  // This will not compile
+```
+
+Mutable variable means that it's value can be changed one or more times:
+
+```kotlin
+var index = 12
+print(index)  // 12
+
+index = 100
+print(index)  // 100
+```
+
+BTW probably you have already noticed that semicolons in Kotlin are not used (they are optional but not needed except few special cases we will explore later in other exercises).
+
+## Functions
+
+Functions in Kotlin are defined with keyword **`fun`** and are [first-class citizens][wiki-fcc] (not related to OOP). It means that you can declare (so-called `top-level functions`) them right in files (e.g. in Java you can define methods only in classes, not in files):
```

Comment:
```suggestion
Functions in Kotlin are defined with the `fun` keyword  and are _first-class citizens_ (not related to OOP). It means that you can declare (so-called `top-level functions`) them right in files (e.g. in Java you can define methods only in classes, not in files):
```

I've suggested a couple of changes:

- Removing the link to the wikipedia page. As per [the docs](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docsintroductionmd): links should be used sparingly, if at all.
- Keywords (and code) are enclosed in backticks. See the [style guide](https://github.com/exercism/v3/blob/master/docs/maintainers/style-guide.md#code)